src/components/Pages/Admin/ToolbarEditor.vue

```diff
@@ -262,6 +271,23 @@ export default {
         showError(message) {
             this.alert = { type: 'danger', message };
         },
+
+        getToolbarItemName(username) {
+            this.$log.debug('Toolbar for username', username);
+            return `toolbar-${username}`;
+        },
+
+        saveToolbarStateToLocalStorage() {
+            this.$log.debug('Saving Toolbar');
+            window.localStorage.setItem(
+                this.getToolbarItemName(this.teacher.username),
+                this.toolbarString,
+            );
+        },
+
+        emptyToolbarString() {
```

Comment:
You have repeated the `emptyToolbarString()` function in two files: consider extracting it into helpers.

Reply:
Done.